pages/itemdetails/_itemId.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div>
+    <v-img :src="imageLink(Item.Id, 'backdrop')"></v-img>
+    <h1>{{ Item.Name }}</h1>
+    <p>{{ Item.Overview }}</p>
+    <v-btn color="primary">Play {{ Item.Type }}</v-btn>
+    <v-btn>More</v-btn>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+export default Vue.extend({
+  data() {
+    return {
+      Name: '',
+      Item: {},
+      Seasons: {},
+      Episodes: []
+    };
+  },
+
+  async beforeMount() {
+    const Item = await this.$itemsApi.getItems({
+      uId: this.$auth.user.Id,
+      userId: this.$auth.user.Id,
+      ids: this.$route.params.itemId,
+      fields: 'Overview'
+    });
+
+    this.Item = Item.data.Items[0];
+  },
+  methods: {
+    imageLink(id: string, type: string) {
```

Comment:
After more than one use, this should turn into a mixin: https://vuejs.org/v2/guide/mixins.html

Reply:
The only reason this is currently a method is because the imageApi isn't working. See [here](https://github.com/jellyfin/jellyfin-vue/pull/29#discussion_r483765025). I can convert it to a mixin now? Or wait till the imageApi is fixed?

Reply:
You can convert it to a mixin now :)

You would just move the entire `methods` part into a Vue mixin and declare it in the component like in normal vue.

Then whenever we want an item's image, we could add the item image mixin to the component and automatically get the result :)

Reply:
I've added it to a mixin. I'm not entirely happy with the mixin/method names though. If you have a better name please suggest 😊.